content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -550,7 +550,7 @@ True or false value.
 
 | Connector     | Default mapping |
 | ------------- | --------------- |
-| PostgreSQL    | `integer`       |
+| PostgreSQL    | `BIGINT`        |
```

Comment:
The capitalization used in this table is the "typical" for each of these databases.

Here an example of the next type down:
```
| Connector     | Default mapping    |
| ------------- | ------------------ |
| PostgreSQL    | `double precision` |
| Microsoft SQL | `float(53)`        |
| MySQL         | `DOUBLE`           |
| MongoDB       | `Double`           |
| SQLite        | `REAL`             |
```

So I would suggest:
```suggestion
| PostgreSQL    | `bigint`        |
```

Reply:
What is "the typical"? Prisma generates Postgres queries in all upper case.

Reply:
While that might be the case (really?), the documentation obviously uses something else here and it would be nice if changes to the docs could adhere to that.

Reply:
Thanks.

Reply:
Thanks @Pwuts and @janpio ! This is kinda a messy one. 

From what I see, the [official Postgres data type is called `bigint`](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-INT), while at Prisma [we map that to our PSL data type called `BigInt`](https://www.prisma.io/docs/concepts/database-connectors/postgresql#native-type-mappings). (And we do use `BigInt` consistently in our docs, when we are talking about Prisma types.) Soit seems that as least as far as Postgres, there is no place for `BIGINT` in our docs (that I see).

If above is accurate, these are the changes we need to make:

1. The edit that is in this PR now, which will replace `integer` with `bigint` in the table called "[Default type mappings"](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#default-type-mappings-3).

2. Do a much better job in the many "mapping" tables that we have in our docs! I think we are overloading the term "native" in the column headers, and sometimes it is unclear if we are talking about a type being native to the DB, or native to Prisma. We need an intro sentence to each table explaining what the columns really are.

Reply:
The Prisma construct that exists (internally) are "native types" or "native database types" (vs. "scalar types"). They express in PSL what PostgreSQL/MySQL/... type should be used in the database. And yes, this is super messy right now and deserves a cleanup.

Reply:
I agree with all of the above. In addition, if consistency is really desired, I suggest looking into how types are mapped from e.g. `BigInt` to `BIGINT` by Prisma internally, since the (PSQL) migration queries Prisma 3.10 generates contain all uppercase types, which is what caused me to apply this to my PR in the first place. The example queries in the Postgres docs have these types in lowercase, in accordance with the documented type names (in lowercase).